rtos/Mutex.h

```diff
@@ -82,7 +82,7 @@ class Mutex : private mbed::NonCopyable<Mutex> {
       Wait until a Mutex becomes available.
 
       @return  status code that indicates the execution status of the function:
-               @a osOK the mutex has been obtained.
+               @a Status code like osOK indicates that the mutex has been obtained.
```

Comment:
"Like" means similar to but not including. "Such as" means similar to and including. So if osOK is a status code that indicates that the mutex has been obtained, please change "like" to "such as". If osOK is similar to status code that indicates that the mutex has been obtained but isn't that, please leave as is.